client/signup/steps/design-picker/index.jsx

```diff
@@ -154,6 +154,14 @@ export default function DesignPickerStep( props ) {
 				result.sort = sortBlogToTop;
 				break;
 		}
+
+		// This is a temporary change until DIFM Lite switches to the full WPCOM theme catalog.
+		// We'll then use the 'difm' intent here.
+		if ( props.useDIFMThemes ) {
+			result.defaultSelection = null;
+			result.sort = sortLocalServicesToTop;
+		}
```

Comment:
Wondering if we should hardcode and provide the DIFM intent dependency from the previous step. So that this is a permanent fix. Not mandatory to address this though.

Reply:
Thanks that makes sense but adding a `difm` intent will be a more significant change, which we can tackle when integrating into the Intent screen. This is a temporary change since the `local-services` category only exists in the custom DIFM categorization and not in the theme catalogue.